articles/virtual-machines/shared-image-galleries.md

```diff
@@ -368,6 +370,16 @@ StorageProfile = new GalleryImageVersionStorageProfile()
                 }
             },
 ```
+
+### Does deleting the Azure Compute Gallery affect VMs (using custom image) created from it?
+VMs created from the Azure Compute Gallery image remains unaffected due to their persistent disks. 
+However, VMSS may fail as they rely on the source image ID reference which would be lost once the Azure Compute Gallery is deleted.
+
+### Does deleting the Azure Compute Gallery in the source region affect VMs created from replicated images (custom images) in the destination region?
+VMs created from the Azure Compute Gallery image remains unaffected due to their persistent disks. 
+However, VMSS may fail as they rely on the source image ID reference which would be lost once the Azure Compute Gallery is deleted.
```

Comment:
Existing VM instances in a scale set are unaffected as the instances are already created and running. However, any scale out operation will fail as new VM instance creation depends on the availability of source image.